app/services/eligibility_service.rb

```diff
@@ -81,9 +81,9 @@ def presence_values
     presence = {}
 
     @client.movements.order(:carrier_date_time).each do |movement|
-      if movement.direction == 'arrival'
+      if movement.arrival?(movement.direction)
```

Comment:
why do we need to pass this is? Can't the `.arrival?` method read that member variable itself?

Reply:
how about just doing:
`if movement.arrival?`

Reply:
@Br3nda Oops! yep got it :)